### DISCUSSION TITLE
PELCO installation

### DISCUSSION THREAD

**Discussion by Katitamorais** [IBEXImagingCommunity/ibex_imaging_knowledge_base#129] [Q&A]
Hello!

First, I would like to share information for new PELCO users. We tried run a cycle of the microwave program without an external chiller and didn't work properly, the temperature was not maintained during the reaction.

Now our lab received the external chiller, however, the water circulation doesn't seem correct. Could you give us any recommendation to operate this equipment?

Thanks.

**Comment by radtkea** [comment]
Hi Katitamorais,

Thank you very much for your question! Excited that you have the PELCO microwave at your institute. 

I am unable to attach a video I recorded showing our set up in the discussion forum. 

The big things are to:
1. Adjust the SteadyTemp Pro "Temp limit" to 50C in your protocol. The default is 26-37C, I believe. If this temperature is exceeded, then the water stops circulating and/or the BioWave might stop. You can edit the Temp Limit using the touchscreen interface when you are designing your protocol. 
2. Make sure the red stripe and clear tubes are properly connected.
3. Make sure you have plenty of water in the system based on the fill line in the little tank in the back.
4. Call tech support. We reached out to them when we first got the system. They were very helpful and recommended the Temp Limit adjustments.

Best of luck and hope this helps!
A

**Comment by Katitamorais** [comment]
Hi @radtkea 

Now we have our Pelco properly installed. However, we have tested three differents temperatures in PELCO for antibody staining, but we got better results using antibody staining 4oC overnight. Have you ever seen something similar?

<img width="765" alt="image" src="https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/163673653/c9a4336c-85c1-4d52-8862-4bc0cd270c7a">


Thanks.
![WhatsApp Image 2024-05-28 at 10 13 46](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/163673653/5d2ef8bd-12a8-46af-8f91-e4968f3d7eff)
![WhatsApp Image 2024-05-28 at 10 13 45](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/163673653/a68097f6-3e08-453e-8990-9064e9480619)
![WhatsApp Image 2024-05-28 at 10 13 45 (1)](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/163673653/baa1f6e6-b132-403c-b543-0a5e07ce7872)

**Reply by radtkea** [reply]
Hi @Katitamorais,

How are things going? Any updates?

Let me know how I can help.

Best wishes,
A

**Comment by Katitamorais** [comment]
@radtkea, below two images demonstrating how we acomodate the humidity chamber inside the PELCO.
![IMG_20240321_103758](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/163673653/3265238a-7b55-4aed-94d7-ae2400af1a76)
![WhatsApp Image 2024-05-28 at 11 54 30](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/163673653/57ed6511-0062-48fb-8522-2c7a84bea2b5)

**Reply by radtkea** [reply]
Sorry to get back to you late. Thank you very much for attaching your photos and explaining the problem!

This is very odd. Everything looks like it is properly set up. I am not sure why it would not be working for you. Are you able to view that the microwave is actually cycling between 0 and 100 Watts? One thought I had was perhaps the instrument is not registering 100 Watts. You should be able to see this if you select "View Graph", I think. Happy to talk some more with you. Best wishes!

**Comment by Katitamorais** [comment]
Hi @radtkea ,

now we are investigating if our PELCO is working properly, I will let you know any updates, ok?

Currently, we are using 4oC for antibody labeling.

Thanks.

**Reply by radtkea** [reply]
Thanks @Katitamorais! 

Keeping fingers crossed for you and your work! Looking forward to positive updates. Happy to talk by phone, if needed.

Have a great week!
A
